### PR TITLE
Set level to info

### LINKED ISSUE #5722: Logs are not being printed
**Describe the bug**
Logs are not printed in the console

**Additional context**
Working on it.

### PR DESCRIPTION
**Pull request by prateekj117**
Fixes #5722 

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by uds5501** [comment]
@prateekj117 setting the level to `INFO` also prints the `Saved to db` and `added to session` type logs. Is it desirable?

**Comment by prateekj117** [comment]
@uds5501 It does help in understanding which end points it is targeting, plus success messages, so it may help in developing, your views on this?

**Comment by uds5501** [comment]
@prateekj117 I am pro this change, although you might want to address the multiple migration error

**Comment by prateekj117** [comment]
@uds5501 On it.

**Comment by prateekj117** [comment]
@iamareebjamal Running python manage.py db heads gives:
Raven is not configured (logging is disabled). Please see the documentation for more information.
Adding job tentatively -- it will be properly scheduled when the scheduler starts
Adding job tentatively -- it will be properly scheduled when the scheduler starts
Adding job tentatively -- it will be properly scheduled when the scheduler starts
Adding job tentatively -- it will be properly scheduled when the scheduler starts
Adding job tentatively -- it will be properly scheduled when the scheduler starts
Added job "sync_events_elasticsearch" to job store "default"
Added job "cron_rebuild_events_elasticsearch" to job store "default"
Added job "send_after_event_mail" to job store "default"
Added job "send_event_fee_notification" to job store "default"
Added job "send_event_fee_notification_followup" to job store "default"
Scheduler started
6e5c574cbfb8 (head)

**Comment by iamareebjamal** [comment]
2 things, INFO messages of libraries should not be printed. Only our INFO messages should be printed.

Secondly, check the return code of the migration head command. It should be 0. If it is non-zero, multiple migration head command will be executed

File: app/__init__.py [hunk 1]
```diff
@@ -100,19 +100,18 @@ def create_app():
...
         'loggers': {
             '': {
                 'handlers': ['console'],
-                'level': logging.WARN
+                'level': logging.INFO
             },
             'default': {
                 'handlers': ['console'],
-                'level': logging.INFO
+                'level': logging.WARN
```
Comment (iamareebjamal) [inline app/__init__.py]: Are requests being logged now?

And only INFO from our application gets printed?

Comment (iamareebjamal) [reply app/__init__.py]: I think opposite will happen. INFO from libraries will get print and from our app won't

Comment (prateekj117) [reply app/__init__.py]: @iamareebjamal Yes, requests are being logged.

**Comment by prateekj117** [comment]
@iamareebjamal Regarding this, I think we should keep the grep change, and discard every other one. That would solve I suppose #5650.

**Pull request closed without merging**